### PR TITLE
TF-3222 Hide reply calendar event action buttons if email is event reply

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Issue
- #3222 

## Reference
- https://www.rfc-editor.org/rfc/rfc5546.html#section-3.2

## Resolved

https://github.com/user-attachments/assets/da2e68ed-3f0a-45e1-8941-906aeaab457e

### CODE REVIEW

File: lib/features/email/presentation/extensions/calendar_event_extension.dart [hunk 1]
```diff
@@ -356,6 +356,7 @@ extension CalendarEventExtension on CalendarEvent {
   }
 
   bool get isDisplayedEventReplyAction => method != null
+    && method != EventMethod.reply
```
Comment (chibenwa) [inline lib/features/email/presentation/extensions/calendar_event_extension.dart:359]: While we are at it, what are other possible values here and for which others does it make sense to hide the buttuns?

Comment (tddang-linagora) [reply lib/features/email/presentation/extensions/calendar_event_extension.dart:359]: Other possible values

Screenshot 2024-10-24 at 09 16 13

Comment (tddang-linagora) [reply lib/features/email/presentation/extensions/calendar_event_extension.dart:359]: IMO

Publish: Hide
Request: Show
Reply: Hide
Add: Show
Cancel: Hide
Refresh: Hide
Counter: Show
DeclineCounter: Hide

@chibenwa How about you?

Comment (chibenwa) [reply lib/features/email/presentation/extensions/calendar_event_extension.dart]: Agree

**Review by dab246 — APPROVED** [review]
LGTM

**Review by hoangdat — APPROVED** [review]

**Pull request merged**